apps/web/data/speakers.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
```

Comment:
In the speakers you can take out a few people, it doesnt make sense to have me and the people responsible for activities such as the Sumo Hacking Challenge.

Reply:
Hmm... The thing is, you're gonna talk, so probably it makes sense to consider you a speaker. About the others, it made sense to me because it's a way to remember who helped that day and also so that the people recognize them in case they need help.

Reply:
I understand why it made sense, but I'm only gonna speak at the beginning of the event, I believe there is no need for me to be put as a Speaker or the people that organize activities, unless they are doing a presentation, like Jess. I believe only the people that will go talk about more relevant things, such as how our Dojo works, about ENSICO and the Roundtable are the main Speakers and the ones that should be the focus on the speaker part of the page

Reply:
Right! I'm going to apply that! :clap: